website/dbt-versions.js

```diff
@@ -18,6 +23,14 @@ exports.versions = [
 ]
 
 exports.versionedPages = [
+  {
+    "page": "docs/reference/dbt-jinja-functions/cross-database-macros",
+    "firstVersion": "1.2",
+  },
+  {
+    "page": "docs/guides/migration-guide/upgrading-to-v1.2",
```

Comment:
I think we need to change this a little:
```suggestion
    "page": "guides/migration/versions/upgrading-to-v1.2",
```
I'll merge and see if it works!